addon/-private/ember-internals.js

```diff
@@ -1,22 +1,25 @@
-import __EMBER_METAL__ from '@ember/-internals/metal/index';
+import * as EMBER_METAL from '@ember/-internals/metal/index';
+
+const POSSIBLE_DECORATORS = ['AliasDecoratorImpl', 'ComputedDecoratorImpl'];
 
 export function getDependentKeys(descriptorOrDecorator) {
-  if (__EMBER_METAL__ && __EMBER_METAL__.descriptorForDecorator) {
-    let descriptor = __EMBER_METAL__.descriptorForDecorator(
-      descriptorOrDecorator
-    );
+  if (EMBER_METAL && EMBER_METAL.descriptorForDecorator) {
```

Comment:
Because the import is stable since ember 3.13 we can remove the fallback:

```js
export function getDependentKeys(descriptorOrDecorator) {
    let descriptor = descriptorForDecorator(descriptorOrDecorator);
    return descriptor._dependentKeys || [descriptor.altKey];
}
```